src/test/java/uk/gov/pay/api/it/CreatePaymentIT.java

```diff
@@ -431,6 +431,7 @@ public void createPayment_responseWith422_whenZeroAmountForDirectDebitAccount()
                 .add("reference", REFERENCE)
                 .add("description", DESCRIPTION)
                 .add("return_url", RETURN_URL)
+                .add("agreement_id", "1234")
```

Comment:
The request should have either a return URL or an agreement ID, not both

Reply:
ah yes 👍